actix-multipart/src/form/json.rs

```diff
@@ -11,35 +11,35 @@ use std::sync::Arc;
 
 /// Deserialize from JSON.
 #[derive(Debug, Deref, DerefMut)]
-pub struct JsonField<T: DeserializeOwned>(pub T);
+pub struct Json<T: DeserializeOwned>(pub T);
 
-impl<T: DeserializeOwned> JsonField<T> {
+impl<T: DeserializeOwned> Json<T> {
     pub fn into_inner(self) -> T {
         self.0
     }
 }
 
-impl<'t, T: DeserializeOwned + 'static> FieldReader<'t> for JsonField<T> {
+impl<'t, T: DeserializeOwned + 'static> FieldReader<'t> for Json<T> {
     type Future = LocalBoxFuture<'t, Result<Self, MultipartError>>;
 
     fn read_field(req: &'t HttpRequest, field: Field, limits: &'t mut Limits) -> Self::Future {
         Box::pin(async move {
-            let config = JsonFieldConfig::from_req(req);
+            let config = JsonConfig::from_req(req);
             let field_name = field.name().to_owned();
 
             let valid_content_type = field.content_type().subtype() == mime::JSON
                 || field.content_type().suffix() == Some(mime::JSON);
 
-            if !valid_content_type && config.content_type_required {
```

Comment:
probably a negligible performance improvement but this could be:
```rust
if config.validate_content_type {
    let valid_content_type = field.content_type().subtype() == mime::JSON
        || field.content_type().suffix() == Some(mime::JSON);
    if !valid_content_type {
```

Reply:
fixed